appservice/src/connectToGitHub.ts

```diff
@@ -9,7 +9,7 @@ import * as opn from 'opn';
 import { Response } from 'request';
 import * as request from 'request-promise';
 import * as vscode from 'vscode';
-import { IAzureNode, IAzureQuickPickItem, IParsedError, parseError, UserCancelledError } from 'vscode-azureextensionui';
+import { AzureTreeItem, IAzureQuickPickItem, IParsedError, parseError, UserCancelledError } from 'vscode-azureextensionui';
```

Comment:
Okay, so for Docker looks like I'll need to create a base class of AzureTreeItem...